pynio/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+    name: pynio
+    version: "1.5.0beta"
+
+source:
+    git_url: https://github.com/NCAR/pynio.git
+    git_tag: master
+
+build:
+    number: 0
+    skip: True  # [win or py3k]
+
+requirements:
+    build:
+        - python
+        - numpy x.x
+        - libpng
+        - jasper
+        - jpeg
+        - zlib
+        - hdf4
+        - hdf5
+        - libnetcdf
+        - proj.4
+        - gdal
+        - curl
+        - libiconv
+        - g2clib
+        - hdfeos2
+        - hdfeos5
+        - gcc
+    run:
+        - python
+        - numpy x.x
+        - libpng
+        - jasper
+        - jpeg
+        - zlib
+        - libnetcdf
+        - hdf5
+        - hdf4
+        - proj.4
+        - gdal
+        - curl
+        - libiconv
+        - g2clib
+        - hdfeos2
+        - hdfeos5
+        - libgcc
```

Comment:
Ping @bekozi and @pelson.

I am starting to think that we should move to conda's `gcc` and `libgcc` to avoid relying too much on pre-installed on system libraries.  What do you think?